### ISSUE TITLE
Regression: 1.4.4 breaks --sig-proxy as nonroot

### ISSUE THREAD

**Issue by edsantiago** [containers/libpod#3492]
`--sig-proxy` seems broken in 1.4.4:
```
$ podman pull alpine
...
$ podman run --tty=false --sig-proxy=true --name foo alpine sh -c 'trap "echo ignoring hup" 1;sleep 20' &
[1] 17452
$ kill -1 $!
$ 
[1]+  Hangup                  podman run --tty=false --sig-proxy=true --name foo alpine sh -c 'trap "echo ignoring hup" 1;sleep 20'
$ podman ps (shows container still running)
```
Expected behavior (as seen in 1.4.2-1.fc30):
```
$ kill -1 $!
$ ... wait 20 seconds, hit ENTER again
ignoring hup
[1]  + done       podman run --tty=false --sig-proxy=true --name foo alpine sh -c
```
podman-1.4.4-1.fc29 and .fc30

**Comment by mheon** [comment]
Did you test 1.4.3, by chance? Trying to determine whether this is just 1.4.4 commits or 1.4.3 as well

**Comment by edsantiago** [comment]
As a matter of fact, yes, I did, but I ignored nonroot errors because of all the hooplah about it being broken in 1.4.3 (a claim I was not actually able to confirm). Same sig-proxy failure on 1.4.3.

**Comment by mheon** [comment]
Thanks, that does narrow it down a lot.

**Comment by rhatdan** [comment]
@edsantiago @mheon Is this fixed in podman 1.5.0?

**Comment by mheon** [comment]
Nope.

If I get a chance this week, I'll look at this, but IMO this is the very bottom of the barrel in terms of priority. `--sig-proxy` is gimmicky to begin with and I don't think people will notice it being broken.

**Comment by mheon** [comment]
@edsantiago Still an issue?

**Comment by edsantiago** [comment]
@mheon as best I can tell this is fixed in podman-1.6.2-2.fc30. Closing. Thank you!
